docker-storage-setup.sh

```diff
@@ -54,12 +54,20 @@ set -e
 #
 # Note: lvm2 version should be same or higher than lvm2-2.02.112 for lvm
 #       thin pool functionality to work properly.
+
+# Check if the script is being run inside of a SPC container by examining $HOST
+# If set, PATH to use $HOST/usr/bin and $HOST/usr/sbin for host helper applications.
```

Comment:
Yeah, but we'll end up loading shared libraries from the host.  And any program that uses absolute paths for helper binaries (not uncommon) will end up using them from the container, not the host...leading to potential confusion.  This seems very possible with lvm, for example in a quick `git grep /sbin` in the LVM git I see:

```
configure.in:                   THIN_CHECK_CMD=/usr/sbin/thin_check
```

The only reliable and sane way to run host binaries to to execute them in the host, i.e. we use the "generate rpm" path for this.